tests/support/modified.conf.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Replace the 3 asterisks with the password -->
+<Passwords>
+    <root_password><![CDATA[***]]></root_password>
+    <sapadm_password><![CDATA[Adm1234]]></sapadm_password>
```

Comment:
As a comment the entry `sapadm_password` is duplicated in the test files. I guess the real file has 2 different entries (`sidadm_password` maybe?)

Reply:
Hi @arbulu89 yes the `sapadm_password` had 2 duplicate entries in the file, sidadm_password is actually the `password` parameter